package.json

```diff
@@ -26,7 +26,7 @@
     "react-dom": "^17.0.2",
     "react-redux": "^7.2.6",
     "react-router-dom": "^6.2.1",
-    "react-scripts": "5.0.0",
+    "react-scripts": "^5.0.0",
```

Comment:
Why did we update this?

Reply:
I wasn't able to run the server and each time I tried to do so, it showed me this error:
"'react-scripts' is not recognized as an internal or external command".
I Managed to get it fixed with: "npm install react-scripts --save".